tests/data/aws/ec2/instances.py

```diff
@@ -509,6 +509,68 @@
             'ReservationId': 'r-03',
         },
     ],
+    'Roles': [

```

Comment:
Is `Roles` a field returned from `aws ec2 describe instances`? If not, please move it to tests/data/aws/iam/.